### PR TITLE
Break out and extend approach

### PR DESCRIPTION
**Pull request by suprenant**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,11 +57,9 @@ NAVAIR HR requires the improvement of both the quality of the data they collect
 
 
 ## Our approach 
-We will begin by experimenting with predictive analytics in the Fleet Readiness Center in Jacksonville, Florida. We then plan to apply learnings and innovatoins to all major Fleet Readiness Centers (three main centers exist).
+We will begin by experimenting with predictive analytics in the Fleet Readiness Center in Jacksonville, Florida. We then plan to apply learnings and innovatoins to all three major FRCs.
```
Comment (tbaxter-18f) [inline README.md]: Typo: Innovations. Dunno how we missed it the first time.

File: approach.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Approach
+
+To help NAVAIR move into the fields of predictive analytics and deep learning, our 18F strategists and engineers will assist NAVAIR with modern data practices aimed at a comprehensive understanding of their HR data elements. This will expand their ability to model data and thus, predict attrition and hiring needs. NAVAIR will then be able to retain necessary skill sets and forecast more efficiently, allowing them to main their fleet and subsequently keep NAVAIR planes in the air. 
+
+In order to do this, we will:
+1. Work collaboratively to review available data to determine the current state of the information
+2. Select a specific data set where we can quickly experiment and learn
+3. Explore the feasibility of modeling it
+4. Develop a preliminary plan/recommendation for how to operationalize/build it out in order to meet NAVAIR’s goals
+
```
Comment (tbaxter-18f) [inline approach.md:10]: I think the section above may be our problem statement? Or could be?

File: approach.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+## Areas of focus
+
+The following five sections represent prioritized areas related to AFRISS-TF that we would like to explore as we determine a path to modernize the system to better meet the needs of its users during this engagement and beyond.
```
Comment (tbaxter-18f) [inline approach.md]: AFRISS-TF?

File: README.md (hunk 1 above)
Comment (suprenant) [reply README.md]: So much for trying to innovate. Good catch, thanks! Fixed.

File: approach.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+## Areas of focus
+
+The following five sections represent prioritized areas related to AFRISS-TF that we would like to explore as we determine a path to modernize the system to better meet the needs of its users during this engagement and beyond. 
+Given our limited time, we almost certainly will not explore any section comprehensively; rather, we will select from these each week as we prioritize our work relative to our capacity.
+ - User research
+ - Data analysis
+ - Systems audit and understanding capabilities (as needed)
+ - Process review (as needed)
+ - Findings and Recommendations
```
Comment (tbaxter-18f) [inline approach.md]: drop the cap on Recommendations since other items are sentence case

File: approach.md (hunk 2 above)
Comment (suprenant) [reply approach.md:10]: That'll come next (probably tomorrow) – I want us to have both what the problem is and how we intend to approach addressing it during this engagement.

File: approach.md (hunk 3 above)
Comment (suprenant) [reply approach.md]: Oop, last project. Sorry for careless mistake, fixed.

Comment (tbaxter-18f) [reply approach.md]: Minor, but you could make this less clause-y: "... related to AFRISS-TF we would to explore as we determine a system modernization path to better meet the needs of users ..."

**Comment by suprenant** [comment]
@tbaxter-18f Fixed the two latest changes you suggested. Take another look for review?

**Review by tbaxter-18f — APPROVED** [review]

**Pull request merged**